chart/values.global.yaml

```diff
@@ -30,6 +30,11 @@ backendRuntime:
 leaderWorkerSet:
   enabled: true
 
+kube-scheduler:
```

Comment:
rename to inftyai-scheduler

Reply:
I can not find it in https://hub.docker.com/u/inftyai?page=1&search=inftyai-scheduler